### ISSUE TITLE
Android 7.1.2

### ISSUE THREAD

**Issue by marksaitis** [easz/doze-tweak#1]
None of this works. Literally spent 12 hours now killing this doze nonsense. Impossible. Settings not getting reset - but most are just not taking effect. The only ones taking effect are - pending idle, where you can specify the amount of time deep maintenance takes. But 0 chance to change how much time it takes for phone to get to deep doze. Takes 35 minutes on mine, every time. Any ideas anyone on how to do a surgery on this thing? Meizu 15 lite with flyme 7 android 7.1.2. I give 100 USD for a working solution.

**Comment by easz** [comment]
to be honest tweaking doze timing is only a small part of the attempt to save battery...

what is your output of `adb shell dumpsys deviceidle`?

**Comment by marksaitis** [comment]
Mate. Im not saving battery. I am only trying to avoid 1 thing - my phone killing wifi when it goes to deep doze. The damn thing is bonkers. I tried many options.... uhhh. On my app I have 3 things implemented - wakelock, wifilock, whitelisted. And the damn thing still kills wifi. I checked to ensure the locks are held using adb.... Phone restarted and all that. I am testing on meizu 15 flyme 7 which is android 7.1.2. I give 200 if somebody can solve this. It's literally pissing me off.
